runtimes/pythonrt/Makefile

```diff
@@ -0,0 +1,44 @@
+ifeq ($(shell uname),Linux)
```

Comment:
can you make the root CI call this when needed? `make pythonrt` comes to mind.

Reply:
IMO it'll be a different CI than the one used for core autokitteh. It'll need Python installed.
If we run everything on the same machine, we'll need to install Python, Node ...

Reply:
sure, but `make pythonrt` is not included in the main ci. it's just an easy way to run this makefile.

Reply:
Done.